src/mo_interpreter/interpret.ml

```diff
@@ -497,21 +516,21 @@ and interpret_exp_mut env exp (k : V.value V.cont) =
           | "put" -> array_put
           | "keys" -> array_keys
           | "vals" -> array_vals
-          | _ -> assert false
+          | s -> invalid_arg s
```

Comment:
This and below were assert false because they should just be unreachable code in type correct programs. But the change is probably fine.

Reply:
This change was useful for finding the original cause of the issue. On the other hand, I suppose it's no longer necessary.